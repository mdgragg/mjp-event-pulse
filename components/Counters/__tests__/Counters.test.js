/**
 * @jest-environment jsdom
 */

import { BoxedCounter } from '../index';
import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';

describe('Counters', () => {
  it('Should render', () => {
    const counter = renderer
      .create(
        <BoxedCounter
          event={{
            eventStartEnd: {
              StartDateTime: new Date('2021-12-31T23:59:59'),
              EndDateTime: new Date('2022-12-31T23:59:59'),
            },
          }}
        />
      )
      .toJSON();
    expect(counter).toMatchSnapshot();
  });
});
