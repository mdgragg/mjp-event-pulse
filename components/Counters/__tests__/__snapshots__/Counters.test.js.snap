// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counters Should render 1`] = `""`;
